foodloop-db-files/seeders/08_insertRestaurant.sql

```diff
@@ -42,7 +42,7 @@ INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Hainanese Delights Menu
 INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Chicken Rice Express Menu');
 INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Chicky Chicky Rice House Menu');
 INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Chicken Master Kitchen Menu');
-INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Rice 'n' Chicken Haven Menu');
+INSERT INTO foodloop.restaurant(restaurantName) VALUES ('Rice ''n'' Chicken Haven Menu');
```

Comment:
nice find! is this verified to correctly escape now? (i am assuming you are trying to escape the `'` value)

Reply:
![image](https://github.com/mahe993/foodloop-server/assets/99639365/d943a657-82a2-4aa9-8b88-050b9bcc6ef4)
yep!

Reply:
oh oops i didn't noticed this because i was checking by looking at color changes